src/GooglePlacesApi.Abstractions/Models/GoogleApiQueryStringParameters.cs

```diff
@@ -15,5 +16,23 @@ public class GoogleApiQueryStringParameters
 
         [AliasAs("types")]
         public string PlaceType { get; set; }
+
+        [AliasAs("sessiontoken")]
+        public string SessionToken { get; set; }
+
+        //[AliasAs("offset")]
```

Comment:
Remove comment :)

Reply:
If not needed, remove the other comments below too

Reply:
This is a more complicated one: the api allows empty strings in the request. Though _offset_, _radius_ and _strictbounds_ cannot be empty. _Offset_ is optional (a different search method). _Radius_ and _strictbounds_ are conditional: they should only be used in combination with _Location_ or _Origin_. 

I've implemented all code to be able to use the  _offset_, _radius_ and _strictbounds_ parameters, though I've commented them out, to keep them out of the API request. Do you have a suggestion how to implement these parameters conditionally?

Reply:
I think I do not get it. I only meant to say: remove the commented code. Leave the attributes as is.

Reply:
I've removed the comments